angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.stories.ts

```diff
@@ -90,8 +90,11 @@ export const wait = () => ({
       modalType="wait"
       idString="modal-wait-1"
     >
-      This type of modal can't be closed by the
-      user.
+      <p class="sprk-u-Width-100">
```

Comment:
hey @bobwise  why is this class needed?

Reply:
I got the solution from this post https://stackoverflow.com/a/35113633/100385